string Class
float64 probability
int64 xmin
int64 ymin
int64 xmax
int64 ymax

int64[] hm_keypoints
float64[] img_keypoints
float64[] probabilities 
float64[] global_measures
float64[] local_measures
